doc/source/dev/governance/governance.rst

```diff
@@ -381,14 +381,13 @@ A list of current Institutional Partners is maintained at the page
 Document history
 ================
 
-https://github.com/numpy/numpy/commits/master/doc/source/dev/governance/governance.rst
+https://github.com/numpy/numpy/blob/master/doc/source/dev/governance/governance.rst
```

Comment:
I think the original link was actually correct here, at least given the section title - it seems that showing the commit history (rather than the current version) was intentional.

Reply:
I agree. This was the one case in the PR where I changed a link that had not been reported bad.  The checker failed with 429's on github, so github checking had to be done by hand.  

Changing it back.